src/util/time.h

```diff
@@ -8,27 +8,34 @@
 
 #include <stdint.h>
 #include <string>
+#include <chrono>
 
 /**
- * GetTimeMicros() and GetTimeMillis() both return the system time, but in
```

Comment:
Need to keep the information here that GetTimeMicros() and GetTimeMillis() aren't mocked

Reply:
Their docstring already says `Returns the system time`. Anything I should add?

Reply:
I think it's better to explicitly mention that they ignore the mocked time. My implicit assumption while reading the documentation would be that mocktime would affect everyting.

Reply:
Yeah, makes sense. Fixed up documentation a bit